pkg/networkserver/redis/registry.go

```diff
@@ -190,11 +191,12 @@ func (m uplinkMatch) IsPending() bool {
 }
 
 type matchKeySet struct {
-	ShortFCnt          string
-	LongFCntNoRollover string
-	LongFCntRollover   string
-	Pending            string
-	Legacy             string
+	ShortFCntLE string
+	LongFCntLE  string
+	Pending     string
+	LongFCntGT  string
```

Comment:
Consider putting `Short` before `Long`, for consistency.

Reply:
The reasoning here is to mimic the sort order